packages/css/src/components/header/header.scss

```diff
@@ -20,7 +20,7 @@
 
 .amsterdam-header__logo {
```

Comment:
I’m tempted to use ‘header logo link’ instead of just ‘header logo’ in the class name and tokens because an outline is expected on a link, not a logo per se, and a link is what it is. Thoughts?

Reply:
I agree!

Reply:
Done!